src/Service.Tests/SqlTests/GraphQLPaginationTests/GraphQLPaginationTestBase.cs

```diff
@@ -886,6 +972,25 @@ public async Task RequestInvalidZeroFirst()
             SqlTestHelper.TestForErrorInGraphQLResponse(result.ToString(), statusCode: $"{DataApiBuilderException.SubStatusCodes.BadRequest}");
         }
 
+        /// <summary>
+        /// Request an invalid number of entries for a pagination page
+        /// </summary>
+        [TestMethod]
+        public async Task RequestInvalidMaxSize()
+        {
+            string graphQLQueryName = "books";
+            string graphQLQuery = @"{
+                books(first: 100001) {
+                    items {
+                        id
+                    }
+                }
+            }";
+
+            JsonElement result = await ExecuteGraphQLRequestAsync(graphQLQuery, graphQLQueryName, isAuthenticated: false);
+            SqlTestHelper.TestForErrorInGraphQLResponse(result.ToString(), statusCode: $"{DataApiBuilderException.SubStatusCodes.BadRequest}");
```

Comment:
substaus code? do you specify a clarifying code that we can associate with invalid `first` value?

Reply:
So I've followed the convention here of other test cases that treat invalid request with substatus code of bad request. This is how it was configured for negative/0 values in other test cases as well.